wic/iot2050-swu-secure.wks.in

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) Siemens AG, 2021-2022
+#
+# Authors:
+#  Quirin Gylstorff <quirin.gylstorff@siemens.com>
```

Comment:
Should be changed to the correct author.

Reply:
Hmm, I felt like I derived it from something you wrote, but I might be right.